advocacy_docs/partner_docs/ToadEdgeGuide/05-UsingToadEdge.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: 'Using Toad Edge'
+description: 'Walkthroughs of example usage scenarios'
+---
+
+Once an instance of EDB Postgres Advanced Server or EDB Postgres Extended is connected to Toad Edge, you can access the capabilities of Toad Edge. 
+
+# Sample User Scenarios
+
+!!! Note
+    This section is provided as an example of Toad Edge and EDB Postgres Advanced Server or EDB Postgres Extended.  It is not intended to show all functionality.
+
+
+
+## Connecting to the Database
+
+1. Launch Toad Edge.
+2. Right click the database connection you want to open and select `Connect`.  You can also select the `Connect` button and select your database.
+
+<p align="center">
+  <img src="Images/ConnectButton.png">
+</p>
+
+3. Enter the username and password and click `OK`.
+4. You are now connected to your database instance.
+
+## Viewing Tables and Data
+
+Using the appropriate SQL code, three tables (`AGENTS`, `CUSTOMERS`, and `ORDERS`) were created in the `edb` database and data was inserted to help visualize some basic functionality. 
+
+<p align="center">
+  <img src="Images/BasicFunctionality.png">
+</p>
+
+1. In the example that is `edb`, you should have an SQL Editor open for your database once you are connected. If you don’t, you can click on the `SQL Editor` button at the top to open a new one.
```

Comment:
Replace
In the example that is `edb`, you should have an SQL Editor open for your database once you are connected.
with
When you connect to the database an SQL Editor area is open for your database.  In this example that is 'edb'.